pkg/odo/cli/url/create.go

```diff
@@ -63,6 +63,9 @@ func (o *URLCreateOptions) Complete(name string, cmd *cobra.Command, args []stri
 	if len(args) == 0 {
 		o.urlName = url.GetURLName(o.Component(), o.componentPort)
 	} else {
+		if len(args[0]) > 63 {
```

Comment:
Shouldn't this check be in `Validate`?

Reply:
@kadel Should it? I don't know, tbh. I can move it to `Validate` function. So, the `Validate` function is called before `Run` function which in turn calls other functions, `Complete` being one of them, which create a URL. But the `Validate` function checks if the user provided data is valid and proceeds further to `Run` only when it's happy. Is that understanding correct?